deny.toml

```diff
@@ -113,6 +113,12 @@ wildcards = "allow"
 highlight = "all"
 # List of crates that are allowed. Use with care!
 
+[[bans.skip]]
+# parking_lot_core 0.9.7 currently depends on this
+# but it will be removed in the next version
+name = "windows-sys"
+version = "0.45"
```

Comment:
First of all, this is unnecessary.  Just update the dependencies which depend on the old version - they were being held back because `parking_lot_core` hadn't yet released a version that use 0.45, but now we can update all of them.  The purpose of this check is to make you go and do those updates in a coordinated way to avoid duplication.  Adding a `skip` should be the last resort, after consultation with others to make sure there isn't a better path forward.

Second of all, the version in the `skip`, if it were necessary, should always be for the old version, not the new version.  And it should always be a precise version.